src/mod/server.mod/tclserv.c

```diff
@@ -582,6 +582,10 @@ static int tcl_server STDVAR {
 
   BADARGS(2, 5, " subcommand ?host ?port? ?password?");
   if (!strcmp(argv[1], "add")) {
+    if (argc < 3) {
+      Tcl_SetResult(irp, "Subcommand add needs at least a server host", TCL_STATIC);
```

Comment:
I would use the error message `wrong # args: should be "server add host ?port? ?password?"`

Reply:
done